Content.Client/Atmos/Consoles/AtmosAlertsComputerBoundUserInterface.cs

```diff
@@ -15,7 +15,7 @@ protected override void Open()
         _menu.OpenCentered();
         _menu.OnClose += Close;
 
-        EntMan.TryGetComponent<TransformComponent>(Owner, out var xform);
+        EntMan.TryGetComponent<TransformComponent>(Owner, out _);
```

Comment:
If this is unused, why are we running `TryGetComponent` at all?

Reply:
Good point